components/symbl_ai/actions/put-speaker-events/put-speaker-events.mjs

```diff
@@ -0,0 +1,35 @@
+import symblAIApp from "../../symbl_ai.app.mjs";
+
+export default {
+  key: "symbl_ai-put-speaker-events",
+  name: "Update Speaker Events",
+  description: "Update the Speaker Events of the Conversation. See the doc [here](https://docs.symbl.ai/docs/conversation-api/speaker-events/).",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    symblAIApp,
+    conversationId: {
+      propDefinition: [
+        symblAIApp,
+        "conversationId",
+      ],
+    },
+    speakerEvents: {
+      type: "string",
+      label: "Speaker Events",
+      description: "Provide a JSON array of speaker events with their `type`, `user` and `offset` information. Each Speaker Event object is represented by the following structure: `[{\"type\": \"started_speaking\",\"user\": {\"id\": \"4194eb50-357d-4712-a02d-94215ead1064\",\"name\": \"user\",\"email\": \"user@example.com\"},\"offset\": {\"seconds\": 0,\"nanos\": 5000000000}},{\"type\": \"stopped_speaking\",\"user\": {\"id\": \"4194eb50-357d-4712-a02d-94215ead1064\",\"name\": \"User\",\"email\": \"user@example.com\"},\"offset\": {\"seconds\": 15,\"nanos\": 5000000000}}]`. See doc [here](https://docs.symbl.ai/docs/conversation-api/speaker-events/#speaker-event-object)",
```

Comment:
Isn't better if you assign this in an array of string like? then parse each element in the array accordingly?
```suggestion
      type: "string[]",
      label: "Speaker Events",
      description: "Provide a speaker event for each row with their `type`, `user` and `offset` information. Each Speaker Event object is represented by the following structure: `{\"type\":\"started_speaking\",\"user\":{\"id\":\"4194eb50-357d-4712-a02d-94215ead1064\",\"name\":\"user\",\"email\":\"user@example.com\"},\"offset\":{\"seconds\":0,\"nanos\":5000000000}}`. See doc [here](https://docs.symbl.ai/docs/conversation-api/speaker-events/#speaker-event-object)",
```

Reply:
@js07 I was trying to avoid the extra parsing by providing the whole array at once. Is this the right way to approach this? (https://github.com/PipedreamHQ/pipedream/blob/040e4fa9e719d3dad20dd05040a361bd8dc1df12/components/shopify/actions/create-order/create-order.mjs#L123)

Reply:
Hey @andrewjschuang, since you are on call this week, could you please provide me some guidance here?

Reply:
Fixed in f99eb7b0.